.github/workflows/main.yaml

```diff
@@ -44,11 +44,12 @@ jobs:
           - [debian11    , "linux/amd64,linux/arm64"]
           - [debian10    , "linux/amd64,linux/arm64"]
           - [debian9     , "linux/amd64,linux/arm64"]
+          - [rocky9      , "linux/amd64,linux/arm64"]
           - [el8         , "linux/amd64,linux/arm64"]
           - [el7         , "linux/amd64,linux/arm64"]
           - [amzn2       , "linux/amd64,linux/arm64"]
-          - [raspbian10  , "linux/arm64"]
-          - [raspbian9   , "linux/arm64"]
+          # - [raspbian10  , "linux/arm64"]
```

Comment:
why remove raspbian?

Reply:
it's slow and we don't build anything for raspbian anywhere